.github/pull_request_template.md

```diff
@@ -21,7 +21,8 @@ If you have made any changes to the database...
   - [ ] I have modified the seed in `supabase/seed/seed.mts` if appropriate
   - [ ] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
   - [ ] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
-  - Does this require resetting the deployed database? - YES / NO (remove as appropriate)
```

Comment:
This line is accounted for by the next bullet point. (and at this point, we should never even allow resetting the database)